appium-tests/common/common.spec.js

```diff
@@ -232,7 +232,13 @@ describe('Contacts Android', function () {
         }
     }
 
-    it('contacts.ui.util configuring driver and starting a session', function (done) {
+    afterAll(function (done) {
+        driver
```

Comment:
Wouldn't hurt to call `checkSession()` here too, because I believe we don't need to call `driver.quit()` if session is not started, and if the driver is undefined, it could lead to the error message like "undefined is not a function".